### PR TITLE
[DOPS-771] Migrate GA jobs to CircleCI

### PR DESCRIPTION
**Pull request by greykurotsuki**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          name: Build gem
+          command: gem build *.gemspec
+      - run:
+          name: Push gem
+          command: gem push *.gem
+
+workflows:
+  version: 2
+
+  sidekiq-group.build-pull-request:
```
Comment (pmatv) [inline .circleci/config.yml:60]: @greykurotsuki, I believe you may use a single workflow for PR and master

**Review by pmatv — CHANGES_REQUESTED** [review]

**Comment by pmatv** [comment]
@greykurotsuki, some checks have been waiting for the status. Please change Git repo settings by specifying proper PR checks.

**Comment by greykurotsuki** [comment]
> @greykurotsuki, some checks have been waiting for the status. Please change Git repo settings by specifying proper PR checks.

Done

**Review by volkorny — APPROVED** [review]

**Review by pmatv — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          name: Push gem
+          command: gem push *.gem
+
+workflows:
+  version: 2
+
+  sidekiq-group.build:
+    jobs:
+
+      - robocop:
```
Comment (pmatv) [inline .circleci/config.yml:63]: Please rename it to `rubocop`

**Review by volkorny — DISMISSED** [review]

**Review by volkorny — APPROVED** [review]

**Pull request merged**